src/civibuild.parse.sh

```diff
@@ -261,6 +261,11 @@ function civibuild_parse() {
         shift
         ;;
 
+      --no-sample-data)
```

Comment:
Whenever a new option is added, we should also describe the option in the help text, `civibuild_app_usage()`.